CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.17 (2020-05-06)
+- Added `updateContext` method to allow child addin to receive updated context information from parent.
```

Comment:
Rephrase to be "Added `updateContext` callback to notify add-in when context information has been updated."

Reply:
should be fixed in latest